src/crl.c

```diff
@@ -99,8 +99,24 @@ static int InitCRL_Entry(CRL_Entry* crle, DecodedCRL* dcrl, const byte* buff,
         return WOLFSSL_FAILURE;
     }
 #endif
-    crle->certs = dcrl->certs;   /* take ownsership */
+#ifdef CRL_USE_CERT_BUFFERS
```

Comment:
Perhaps there is a better name for `CRL_USE_CERT_BUFFERS`? This ends up using a fixed (static) list of RevokedCerts to allow for the binary search instead of allocated ones. Perhaps a better name is `CRL_STATIC_REVOKED_LIST` to pair with the `CRL_MAX_REVOKED_CERTS`?

Reply:
Also please document the new macro at the top of crl.c (like we did for asn.c). Thanks